curator/patterns/post-navigation.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Post Navigation
+ * Slug: curator/post-navigation
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px","bottom":"30px"}},"border":{"top":{"color":"var:preset|color|foreground","width":"1px"},"bottom":{"color":"var:preset|color|foreground","width":"1px"}}},"className":"alignfull","layout":{"inherit":false}} -->
+<div class="wp-block-group alignfull" style="border-top-color:var(--wp--preset--color--foreground);border-top-width:1px;border-bottom-color:var(--wp--preset--color--foreground);border-bottom-width:1px;padding-top:30px;padding-bottom:30px"><!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:columns {"isStackedOnMobile":false} -->
+<div class="wp-block-columns is-not-stacked-on-mobile"><!-- wp:column {"width":"50%","layout":{"inherit":true}} -->
+<div class="wp-block-column" style="flex-basis:50%"><!-- wp:group {"style":{"spacing":{"blockGap":"2px"}},"className":"curator-post-navigation","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group curator-post-navigation"><!-- wp:paragraph {"className":"curator-post-navigation-icon","fontSize":"large"} -->
+<p class="curator-post-navigation-icon has-large-font-size">←</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:post-navigation-link {"type":"previous","label":"<?php echo esc_html__( 'PREVIOUS POST', 'curator' ); ?>","style":{"typography":{"lineHeight":"1.5","textTransform":"uppercase"}},"fontSize":"large"} /--></div>
+<!-- /wp:group --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:group {"style":{"spacing":{"blockGap":"2px"}},"className":"curator-post-navigation","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"right"}} -->
+<div class="wp-block-group curator-post-navigation"><!-- wp:post-navigation-link {"label":"<?php echo esc_html__( 'NEXT POST', 'curator' ); ?>","style":{"typography":{"textTransform":"uppercase"}},"fontSize":"large"} /-->
```

Comment:
icons ← and → font not matching. also the gap between icon and text.

<img width="218" alt="image" src="https://user-images.githubusercontent.com/1935113/169805600-ba9972e2-41a7-45d6-a2ef-23f3fa9ba8ec.png">
<img width="218" alt="image" src="https://user-images.githubusercontent.com/1935113/169805851-13222563-ffd5-4dba-aa92-17ba817b5f57.png">

Just icon appears when there isn't next/prev post.
<img width="551" alt="image" src="https://user-images.githubusercontent.com/1935113/169806114-0d048be0-6f5d-4fd7-8647-6f6970818698.png">
May be it is safe to keep the arrow part of label instead of a separate block.

Reply:
> May be it is safe to keep the arrow part of label instead of a separate block

I agree with this.